src/Components/NumberButton.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+function NumberButton({ number, onClick }) {
+  return (
+    <button name={number} type="button" onClick={onClick}>
```

Comment:
App 컴포넌트에서 buttonClick함수에서 event.target.name 값을 사용야하는 이유가 있을까요? 이렇게 숫자만 넘기도록 처리하는건 어떤가요?

```suggestion
    <button type="button" onClick={() => onClick(number)}>
```